.changeset/popular-rats-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"@turnkey/sdk-browser": patch
```

Comment:
nit: in semver this should be a minor change (new feature), not a patch (which would be a "bug fix")